CIScripts/Test/GetTestConfigurationJuni.ps1

```diff
@@ -0,0 +1,67 @@
+function Get-TestConfiguration {
+    [TestConfiguration] @{
+        ControllerIP = "10.7.0.54";
+        ControllerRestPort = 8082;
+        ControllerHostUsername = "ubuntu";
+        ControllerHostPassword = "ubuntu";
+        AdapterName = "Ethernet1";
+        VMSwitchName = "Layered Ethernet1";
```

Comment:
This used to be ` VMSwitchName = "Layered?Ethernet1";`, why the change to use a space instead of question mark?

Reply:
The question mark was introduced https://github.com/codilime/contrail-windows-ci/commit/0e52ce06dc0c9f0e369f2bca2414eabba7fd9ed3 but it was then reverted. So the question mark is no longer needed